app/models/review.rb

```diff
@@ -0,0 +1,9 @@
+class Review < ApplicationRecord
+  belongs_to :art, dependent: :destroy
+  belongs_to :user
+  belongs_to :booking
+
+  validates :title, presence: true
+  validates :comment, presence: true, length: { minimum: 5 }
```

Comment:
I think leaving an empty review is completely fine.
If you go on Amazon there are thousands of reviews but only a few have a comment for example.